graph-commons/src/test/scala/io/renku/graph/http/server/security/DatasetIdRecordsFinderSpec.scala

```diff
@@ -51,10 +49,11 @@ class DatasetIdRecordsFinderSpec
       )
     }
 
-    "return SecurityRecord with project visibility, path and no member is project has none" in new TestCase {
+    "return SecurityRecord with project visibility, path and no member if project is none" in new TestCase {
```

Comment:
"... project is none" -> "... project has none"? not sure :) or perhaps "... has no member"